bigbluebutton-html5/imports/ui/components/common/meeting-remaining-time-graphql/queries.ts

```diff
@@ -0,0 +1,18 @@
+import { gql } from '@apollo/client';
+
+export type breakoutDataResponse = {
+  breakoutRoom: Array<{durationInSeconds: number, startedAt: string}>;
+}
+
+export const FIRST_BREAKOUT_DURATION_DATA_SUBSCRIPTION = gql`
+  subscription firstBreakoutData {
```

Comment:
The meeting time doesn't have the same data?

Reply:
No. The meeting time is for the current meeting (main room or breakout). When accessing the breakout panel, it shows the breakout remaining time, not the remaining time of the current meeting:
![image](https://github.com/bigbluebutton/bigbluebutton/assets/56703993/f451a68e-e72d-463f-aed6-9a545e48a2a2)